src/UI/Components/Diagram/actions.ts

```diff
@@ -479,13 +537,17 @@ export function appendColumns(
   attributesKeywords: string[],
   serviceInstanceAttributes: InstanceAttributeModel,
   isInitial = true,
+  isEmpty = false, // TODO: temporary solution to handle empty values until right sidebar is finished
 ) {
   const instanceAttributes = {};
   const attributes = attributesKeywords.map((key) => {
-    instanceAttributes[key] = serviceInstanceAttributes[key];
+    const attributeValue = isEmpty
+      ? ""
+      : (serviceInstanceAttributes[key] as string);
```

Comment:
why the casting here? can't you type the key in line 543?

Reply:
changed for String(serviceInstanceAttributes[key])